app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -7,50 +7,151 @@ class RefereeProfile extends Component {
     match: PropTypes.shape({
       params: PropTypes.object
     }).isRequired
-  }
+  };
 
   state = {
+    httpStatus: 200,
+    httpStatusText: '',
     firstName: '',
     lastName: '',
     bio: '',
-    email: ''
-  }
+    email: '',
+    pronouns: '',
+    nationalGoverningBodies: [],
+    certifications: []
+  };
 
   componentDidMount() {
-    const { match: { params } } = this.props
+    const { match: { params } } = this.props;
 
     axios.get(`/api/v1/referees/${params.id}`)
-      .then(({ data }) => {
-        const { attributes } = data.data
+      .then(({ status, statusText, data }) => {
+        const { data: { attributes }, included } = data;
+        const certifications = included
+          .filter(({ type }) => type === 'certification')
+          .map(certification => certification.attributes);
+        const nationalGoverningBodies = included
+          .filter(({ type }) => type === 'national_governing_body')
+          .map(nationalGoverningBody => nationalGoverningBody.attributes);
         this.setState({
+          httpStatus: status,
+          httpStatusText: statusText,
           firstName: attributes.first_name,
           lastName: attributes.last_name,
           bio: attributes.bio,
-          email: attributes.email
+          email: attributes.email,
+          pronouns: attributes.pronouns,
+          nationalGoverningBodies,
+          certifications
         })
       })
+      .catch(({ response: { status, statusText } }) => {
+        this.setState({
+          httpStatus: status,
+          httpStatusText: statusText
+        });
+      })
   }
 
   render() {
     const {
+      httpStatus,
+      httpStatusText,
       firstName,
       lastName,
       bio,
-      email
-    } = this.state
+      email,
+      pronouns,
+      nationalGoverningBodies,
+      certifications
+    } = this.state;
+
+    if (httpStatus !== 200) {
+      return (
+        <h1>
+          {httpStatusText}
+        </h1>
+      )
+    }
 
     return (
       <div>
-        <h1>Referee Profile</h1>
-        <h3>
-          {`Name: ${firstName} ${lastName}`}
-        </h3>
-        <h3>
-          {`Bio: ${bio}`}
-        </h3>
-        <h3>
-          {`Email: ${email}`}
-        </h3>
+        <h1>
+          {`${firstName} ${lastName}`}
+        </h1>
+        <dl>
+          {pronouns
+            && (
+              <Fragment>
+                <dt>
+                  Pronouns:
+                </dt>
+                <dd>
+                  {pronouns}
+                </dd>
+              </Fragment>
+            )
+          }
+          <dt>
+            National Governing
+            {nationalGoverningBodies.count > 1 ? ' Bodies' : ' Body'}
+            :
+          </dt>
+          {
+            nationalGoverningBodies
+              .map(
```

Comment:
would be nice to have this as a small render function, something like:

```
renderNGB = (nationalGoverningBody) => {
  return (
    <dd key={nationalGoverningBody.name}>
       <a>{nationalGoverningBody.name}</a>
    </dd>
  )
}
```

so line 101 can become `{ nationalGoverningBodies.map(this.renderNGB) }`